CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.31.0]
+
+## What's Changed
+* Add run_command benchcomp visualization by @karkhaz in https://github.com/model-checking/kani/pull/2542
```

Comment:
`benchcomp` contributions shouldn't appear here, right? cc @karkhaz

Reply:
let me remove it for now, we can patch it later, for the actual release